File: dart2/PHP/Dart2LexerBase.php
<?php

//declare(strict_types=1);

use Antlr\Antlr4\Runtime\Lexer;

abstract class Dart2LexerBase extends Lexer
{
        public function __constructor($input)
        {
                parent::_construct($input);
        }

        public function CheckNotOpenBrace(): bool
        {
                return $this->input->LA(1) != '{';
        }
}

File: dart2/Go/dart2_lexer_base.go
package parser

import (
    "github.com/antlr4-go/antlr/v4"
)

type Dart2LexerBase struct {
    *antlr.BaseLexer
}

func (l *Dart2LexerBase) CheckNotOpenBrace() bool {
    return l.GetInputStream().(antlr.CharStream).LA(1) != '{';
}
